File: frontend/src/models/History.ts
export interface HistoryItem {
  id: number;
  name: string;
  created_at: string;
  folder_path: string;
  cctv_list?: string[];
  var_threshold: number;
  learning_rate: number;
  epoch: number;
}

export interface HistoryResponse {
  results: HistoryItem[];
}

File: backend/src/features/parking/model/response/history.go
package response

type ResHistory struct {
	Results []HistoryItem `json:"results"`
}

type HistoryItem struct {
	Id           int      `json:"id"`
	Name         string   `json:"name"`
	CreatedAt    string   `json:"created_at"`
	FolderPath   string   `json:"folder_path"`
	Epoch        int      `json:"epoch"`
	LearningRate float64  `json:"learning_rate"`
	VarThreshold float64  `json:"var_threshold"`
	CctvList     []string `json:"cctv_list"`
}
